### PR TITLE
[readme] Document official mission (closes #3)

### LINKED ISSUE #3: Charter
We have a semblance of a charter floating around in various meeting notes, e.g.

https://docs.google.com/document/d/1I0zvGPp245L3ZBtdvl4UXrLqiFGZ7pY-q9pjcVxkHCU/edit?usp=sharing

It'd be good to write it down and post it somewhere, e.g. in the README.md of this repo

### PR DESCRIPTION
**Pull request by joshlf**
Let's use this PR as a place to workshop the wording of our mission and goals. The current version of this PR is taken from the [original announcement](https://internals.rust-lang.org/t/proposal-security-working-group/8282/134?u=joshlf), and should be thought of merely as a first draft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 
 [issue tracker]: https://github.com/rust-secure-code/wg/issues
 
+## Mission
+
+Our mission is to make it easy to write secure code in Rust.
+
+We have the following goals for the Rust language and ecosystem:
+- Most tasks shouldn't require dangerous features such as `unsafe`. This includes FFI.
+- Mistakes in security code should be easily caught.
```
Comment (alex) [inline README.md]: Maybe call out that this includes with both static analysis and human review?

Comment (joshlf) [reply README.md]: How about:

> Mistakes in security code should be easily caught by machines or, failing that, humans aided by machines.

Comment (alex) [reply README.md]: LGTM, thanks!

File: README.md [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 
+## Mission
+
+Our mission is to make it easy to write secure code in Rust.
+
+We have the following goals for the Rust language and ecosystem:
+- Most tasks shouldn't require dangerous features such as `unsafe`. This includes FFI.
+- Mistakes in security code should be easily caught by machines or, failing
+  that, humans aided by machines.
+- It should be clear how to programmers how to use security-sensitive APIs
```
Comment (zachreizner) [inline README.md]:
```suggestion
- It should be clear to programmers how to use security-sensitive APIs
```

**Comment by stusmall** [comment]
Should we expand:
"It should be clear how to programmers how to use security-sensitive APIs"
to 
"It should be clear how to programmers how to perform security-sensitive tasks"

This will include the original goal and expand it to include common tasks password storage, authentication, etc instead of just APIs

File: README.md (hunk 2 above)
Comment (joshlf) [reply README.md]: Done.

**Comment by joshlf** [comment]
@stusmall Done.

**Comment by joshlf** [comment]
FCP, so to speak: I'll merge this later today unless there are any objections.

**Comment by tarcieri** [comment]
@joshlf I YOLO'd my only suggestion myself 😉

**Pull request merged**